lib/simplecov_default_profile.rb

```diff
@@ -0,0 +1,11 @@
+SimpleCov.profiles.define 'default' do
```

Comment:
I think this should go in the `spec` folder, given that it's only usable by the test environment.

Reply:
Alright!